src/Satooshi/Bundle/CoverallsV1Bundle/Entity/Exception/RequirementsNotSatisfiedException.php

```diff
@@ -24,6 +24,32 @@ class RequirementsNotSatisfiedException extends \RuntimeException
     protected $readEnv;
 
     /**
+     * Array of secret env vars.
+     *
+     * @const array
+     */
+    private static $secretEnvVars = array(
+        'COVERALLS_REPO_TOKEN',
+    );
+
+    /**
+     * Format a pair of the envVarName and the value.
+     *
+     * @param string $key   the env var name.
+     * @param string $value the value of the env var.
+     *
+     * @return string
+     */
+    protected function format($key, $value)
+    {
+        if (in_array($key, self::$secretEnvVars, true)) {
+            $value = preg_replace('/./', '*', $value) . '(REDUCTED)';
```

Comment:
I get the intention but didn't know that word, in fact I didn't found it in dict.
Glad of your change